### DISCUSSION TITLE
英文冒号编译不出来

### DISCUSSION THREAD

**Discussion by Martian9119** [tuna/thuthesis#869] [Q&A]
我在输入数学公式的时候，英文的冒号编译后就消失了。比如我输入“函数f: X→I”，编译出来变成了“函数f X→I”。

**Comment by note286** [comment]
你的问题无法复现，Windows 10，XeLaTeX。

```latex
\documentclass{thuthesis}
\begin{document}
函数$f:X\rightarrow I$
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/235280349-f52b2451-274f-4427-8f56-7c1d5f484093.png)
